openupgrade_scripts/scripts/hr_attendance/18.0.2.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,29 @@
+---Models in module 'hr_attendance'---
+---Fields in module 'hr_attendance'---
+hr_attendance / hr.attendance            / expected_hours (float)        : NEW isfunction: function, stored
+hr_attendance / hr.attendance            / in_mode (selection)           : selection_keys is now '['kiosk', 'manual', 'systray', 'technical']' ('['kiosk', 'manual', 'systray']')
+hr_attendance / hr.attendance            / out_mode (selection)          : selection_keys is now '['auto_check_out', 'kiosk', 'manual', 'systray', 'technical']' ('['kiosk', 'manual', 'systray']')
+hr_attendance / hr.attendance            / overtime_status (selection)   : NEW selection_keys: ['approved', 'refused', 'to_approve'], isfunction: function, stored
+hr_attendance / hr.attendance            / validated_overtime_hours (float): NEW hasdefault: compute
```

Comment:
precreate `overtime_status` and fill it with `approved`
precreate `validated_overtime_hours` and fill it with `overtime_hours`